testsupport/nexus-pax-exam/src/main/java/org/sonatype/nexus/pax/exam/NexusPaxExamSupport.java

```diff
@@ -74,9 +74,9 @@
  * <pre>
  * &#064;Configuration
  * public static Option[] config() {
- *   return options( //
- *       nexusDistribution(&quot;org.sonatype.nexus.assemblies&quot;, &quot;nexus-base-template&quot;), //
- *       nexusPlugin(&quot;org.sonatype.nexus.plugins&quot;, &quot;nexus-yum-repository-plugin&quot;) //
+ *   return options(
+ *       nexusDistribution(&quot;org.sonatype.nexus.assemblies&quot;, &quot;nexus-base-template&quot;),
+ *       nexusPlugin(&quot;org.sonatype.nexus.plugins&quot;, &quot;nexus-repository-raw&quot;)
```

Comment:
might want to pick one that's not included by default, as the above comment suggests?